HACKING.md

```diff
@@ -184,3 +186,29 @@ tag if it was created.
 * Flutter is currently a fork to add support for line numbers. We need to
   figure out a cleaner way of doing this so we can merge it into the
   upstream codebase.
+
+## Glossary
+
+<dl>
+  <dt>reStructuredText</dt>
```

Comment:
[minor nit] it would be nice to have links to official API documentation for anywhere we are using an external source ie. https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html

Reply:
We technically support The Good Parts of reStructuredText, not the entire thing. It's a recurring problem that people go to that page, and then try to use misfeatures that should not exist.

Unfortunately, we don't have authoritative documentation on what we do support, but I would rather not direct people to a known bad source.